app/controllers/api/v4/application_controller.rb

```diff
@@ -88,7 +88,7 @@ def validate_user
   end
 
   def set_race
-    @race = Race.find(params[:race])
+    @race = Race.friendly_find(params[:race])
```

Comment:
I thought we were going to have the API take full ID's only?

Reply:
Oh oops, I don't think I realized this was an API controller. I'll revert.